content/about/community/partners/index.md

```diff
@@ -19,7 +19,7 @@ Istio benefits from an ecosystem of great partners. Check 'em out and learn how
     {{< company_logo link="https://www.datadoghq.com" logo="./datadog.png" alt="Datadog" tagline="Modern monitoring & analytics" >}}
     {{< company_logo link="https://www.envoyproxy.io" logo="./envoy.svg" alt="Envoy" tagline="An open source edge and service proxy, designed for cloud-native applications" >}}
     {{< company_logo link="https://cloud.google.com" logo="./gcp.svg" alt="Google Cloud Platform" tagline="Make your next move here" >}}
-    {{< company_logo link="https://www.ibm.com" logo="./ibm.svg" alt="IBM" >}}
+    {{< company_logo link="https://www.ibm.com/cloud/" logo="./ibmcloud.png" alt="Built for all your applications. AI ready. Secure to the core." >}}
```

Comment:
alt should be "IBM Cloud", and your sentence should be used with tagline="" instead.

You can't find an SVG of the graphic? PNGs end up not looking that good when using the dark theme (select that from the gear menu to see)

Also, there's a link from the Chinese translation to the image, so you'll need to patch that up too.

Reply:
Done, thanks @geeknoid